frontend/src/components/Question.js

```diff
@@ -126,11 +127,19 @@ class Question extends Component {
   /**
    * void function
    * controls when to show result of user's chosen answer and the correct answer
+   * sets correctness of this question's answer choice
    */
   submitClick = () => {
-    if (this.state.submitted === false && this.state.selectedAnswer !== -1) {
+    if (!this.state.submitted && this.state.selectedAnswer !== -1) {
+      let correctAnswer;
```

Comment:
Can we change this to `answeredCorrectly`? I tend to understand this as "this variable represents the correct answer"